pulp_container/app/access_policy.py

```diff
@@ -24,15 +28,14 @@ def has_namespace_obj_perms(self, request, view, action, permission):
             namespace = obj.namespace
             return request.user.has_perm(permission, namespace)
         elif type(obj) == models.ContainerPushRepository:
-            dists_qs = models.ContainerDistribution.objects.filter(repository=obj)
-            for dist in dists_qs:
+            for dist in obj.distributions:
```

Comment:
nice

Reply:
Except it didn't really work that way...